src/main/webapp/WEB-INF/jspf/header.jspf

```diff
@@ -6,6 +6,13 @@
         <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
         <meta http-equiv="Content-Language" content="en" />
         <meta name="robots" content="noindex,nofollow" />
+        <link rel="icon" href="https://avatars2.githubusercontent.com/u/7898027?s=200&v=4" />
```

Comment:
Is this good idea, why not store the icon locally

Reply:
will do